server/models/PetType.js

```diff
@@ -15,14 +15,41 @@ class PetType {
       const result = await pool.query("SELECT * FROM pet_types;")
       const typesData = result.rows
       const types = typesData.map(type => new this(type))
-
       return types
+    } catch (error) {
+      console.log(error)
+      throw (error)
+    }
+  }
 
+  async pets() {
+    const petFile = await import('./Pet.js')
+    const Pet = petFile.default
+    try {
+      const query = "SELECT * FROM adoptable_pets WHERE pet_type_id = $1;"
+      const result = await pool.query(query, [this.id]) 
+      const petsData = result.rows
+      const pets = petsData.map(pet => new Pet(pet))
+      return pets
     } catch (error) {
       console.log(error)
       throw (error)
     }
   }
+
+  static async findPetType(type) {
```

Comment:
Might consider renaming this method `findByPetType` or `findByType`

Reply:
since this returns a PetType object, I think that findPetType is an ok name. Does it make sense?

Reply:
`findPetType` to me reads as "find _the_ pet type", renaming to something like `findByPetType` or `findByType` is similar to other methods we create like `findById`